habitat_baselines/run.py

```diff
@@ -33,18 +33,33 @@ def main():
         nargs=argparse.REMAINDER,
         help="Modify config options from command line",
     )
+
     args = parser.parse_args()
-    config = get_config(args.exp_config, args.opts)
+    run_exp(**vars(args))
+
+
+def run_exp(exp_config: str, run_type: str, opts=None) -> None:
+    r"""Runs experiment given mode and config
```

Comment:
Usually `main` method is the last one in the file:
```suggestion
    r"""Runs an experiment with given mode and config.
```